recipes/lely-core/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.2)
+project(test_package)
+
+find_package(lely-core REQUIRED)
+find_package(Threads REQUIRED)
+
+add_executable("${PROJECT_NAME}" test_package.cpp)
+target_link_libraries("${PROJECT_NAME}" PRIVATE lely-core::lely-core Threads::Threads)
```

Comment:
pthread should be added as a system library in the recipe

Reply:
pthread is specified as a system library at line 229 in conanfile.py, i removed the `Threads::Threads` linking